.tool-versions

```diff
@@ -1 +1 @@
-updatecli v0.95.1
\ No newline at end of file
+updatecli v0.104.0
```

Comment:
introduced a hard dependency with the updatecli version, see [updatecli/policies@main/updatecli/policies/autodiscovery/updatecli/CHANGELOG.md#090](https://github.com/updatecli/policies/blob/main/updatecli/policies/autodiscovery/updatecli/CHANGELOG.md#090)